janusgraph-test/pom.xml

```diff
@@ -100,13 +100,13 @@
             <plugin>
                 <groupId>org.codehaus.gmavenplus</groupId>
                 <artifactId>gmavenplus-plugin</artifactId>
-                <version>1.2</version>
+                <version>1.5</version>
                 <executions>
                     <execution>
                         <goals>
                             <goal>addSources</goal>
                             <goal>addTestSources</goal>
-                            <goal>generateStubs</goal>
+                            <goal>testGenerateStubs</goal>
```

Comment:
This one I messed up, and shouldn't have renamed that goal, as the testGenerateStubs one is already mentioned lower down, Thanks for checking.